tests/utils.py

```diff
@@ -27,6 +27,20 @@ def run_process(process_code, process_name, example_input):
     )
 
 
+def run_process_with_additional_js_code(
```

Comment:
Looks a lot better now. However, I think it could be improved a bit further. However, I think it makes a bit less clear that we have separate `example_input` and `additional_params_in_string` (and the name isn't very descriptive either). I think the entire arguments string could also just be passed to this function.

Reply:
I agree, this is an even cleaner solution. Changed.